pkg/arm/data/master-startup.sh

```diff
@@ -689,6 +689,9 @@ restorecon -Rv /var/lib/etcd
 logger -t master-startup.sh "restore done"
 {{- end }}
 
+#set the recommended openshift-control-plane profile on first boot
+tuned-adm profile openshift-control-plane
```

Comment:
is tuned-adm available on the nodes by default?

Reply:
Yes, it's provided as part of the tuned package. And tuned is installed in our images.